packages/1xdev-random-number/__test__/tsconfig.json

```diff
@@ -9,5 +9,7 @@
     "*.ts",
     "**/*.tsx",
     "**/*.ts"
-  ]
+  ],
+  "types": ["node", "jest", "@testing-library/jest-dom"],
```

Comment:
This should already be part of the `@1xdev/tsconfig/jest.json` right?

Reply:
Let me check that